app/views/users/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="top-space">
+  <div class="container">
+    <h1 class="text-center be-light-blue">USERS</h1>
+    <table class="table">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Last Name</th>
+          <th scope="col">Email</th>
+          <th scope="col">CellPhone</th>
+          <th scope="col">Address</th>
+          <th scope="col">Show</th>
+          <!--<th scope="col">Delete</th>-->
+        </tr>
+      </thead>
+      <tbody>
+
+          <% @users.each do |user| %>
+            <tr>
+              <th scope="row"><%= user.id %></th>
+              <td><%= user.first_name %></td>
+              <td><%= user.last_name %></td>
+              <td><%= user.email %></td>
+              <td><%= user.cellphone %></td>
+              <td><%= user.address %></td>
+              <td><button type="button"
+                          class="btn btn-success" ><a href="/users/<%= user.id%>"
+                                                      style="color: white;
+                                                            text-decoration: none;">Show
+                                                    </a>
+              </button></td>
+              <!--<td><button type="button" class="btn btn-danger">Delete</button></td>-->
```

Comment:
You should use the rails helper for HTML tags. example https://apidock.com/rails/ActionView/Helpers/FormTagHelper/button_tag

Also for the link.